views/js/submitScore.js

```diff
@@ -0,0 +1,18 @@
+var scorePopUp = function () {
+  var name = prompt("Please enter your name:", "anonymous");
+  var score = getScore();
+  console.log(score);
+  const xhr = new XMLHttpRequest();
+  const data = `name=${name}&score=${score}`
```

Comment:
You might also want to send `application/json` data rather than `application/x-www-form-urlencoded` data.

Reply:
Done in a353ae6